sdk/textanalytics/ai-text-analytics/src/generated/generatedClient.ts

```diff
@@ -1,211 +1,154 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
  *
  * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is
- * regenerated.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
 import * as coreHttp from "@azure/core-http";
+import * as Parameters from "./models/parameters";
 import * as Models from "./models";
 import * as Mappers from "./models/mappers";
-import * as Parameters from "./models/parameters";
 import { GeneratedClientContext } from "./generatedClientContext";
+import {
+  GeneratedClientOptionalParams,
+  MultiLanguageBatchInput,
+  GeneratedClientEntitiesRecognitionGeneralOptionalParams,
+  GeneratedClientEntitiesRecognitionGeneralResponse,
+  GeneratedClientEntitiesLinkingOptionalParams,
+  GeneratedClientEntitiesLinkingResponse,
+  GeneratedClientKeyPhrasesOptionalParams,
+  GeneratedClientKeyPhrasesResponse,
+  LanguageBatchInput,
+  GeneratedClientLanguagesOptionalParams,
+  GeneratedClientLanguagesResponse,
+  GeneratedClientSentimentOptionalParams,
+  GeneratedClientSentimentResponse
+} from "./models";
 
 class GeneratedClient extends GeneratedClientContext {
   /**
    * Initializes a new instance of the GeneratedClient class.
    * @param endpoint Supported Cognitive Services endpoints (protocol and hostname, for example:
-   * https://westus.api.cognitive.microsoft.com).
-   * @param [options] The parameter options
+   *                 https://westus.api.cognitive.microsoft.com).
```

Comment:
@joheredi, @sarangan12,  

I know we talked about this in https://github.com/Azure/azure-sdk-for-js/pull/10100#discussion_r456593037, but did we check how this shows up in the editor suggestions? If the empty space shows up there, then it is not a good idea

Reply:
The editor suggestions seem to ignore the extra spaces

This is without the spaces:
![image](https://user-images.githubusercontent.com/21109913/88337320-64475a00-cceb-11ea-911f-6232a356e815.png)


This is with the spaces:
![image](https://user-images.githubusercontent.com/21109913/88337420-922c9e80-cceb-11ea-8523-5ae2c462087f.png)

Reply:
Is there any formatter that can enforce this?
The reason I ask is that the convenience layer we work on will not have the similar treatment because when typing as human we would generally just press enter at the end of the line and keep typing. So now, we end up with generated code following a formatting that cannot be enforced in the convenience layer

Regarding the editor, can we try on editors other than VS Code and see how it shows up there?

Reply:
Unfortunately I haven't been able to find a way to enforce this on the formatter. 

I agree with having consistency and if the formatter cannot enforce this, it will be very difficult to achieve consistency with convenience layer. I have added this to Azure/autorest.typescript/issues/550